setup.py

```diff
@@ -89,18 +89,13 @@
             "prepare_task_data=nemo_curator.scripts.prepare_task_data:console_script",
             "find_matching_ngrams=nemo_curator.scripts.find_matching_ngrams:console_script",
             "remove_matching_ngrams=nemo_curator.scripts.remove_matching_ngrams:console_script",
-            "gpu_compute_minhashes=nemo_curator.scripts.compute_minhashes:console_script",
-            "minhash_buckets=nemo_curator.scripts.minhash_lsh:console_script",
-            "jaccard_map_buckets=nemo_curator.scripts.map_buckets:console_script",
-            "jaccard_shuffle=nemo_curator.scripts.jaccard_shuffle:console_script",
-            "jaccard_compute=nemo_curator.scripts.jaccard_compute:console_script",
-            "gpu_connected_component=nemo_curator.scripts.connected_components:console_script",
-            "write_deduped_result_with_text=nemo_curator.gpu_deduplication.write_deduped_result_with_text:console_script",
-            "verify_all_pairs_jaccard=nemo_curator.gpu_deduplication.verify_all_pairs_jaccard:console_script",
+            "gpu_compute_minhashes=nemo_curator.scripts.fuzzy_deduplication.compute_minhashes:console_script",
```

Comment:
I get this error when trying to run the scripts:
```
Traceback (most recent call last):
  File "/home/rywolf/.local/bin/minhash_buckets", line 5, in <module>
    from nemo_curator.scripts.fuzzy_deduplication.minhash_lsh import console_script
ModuleNotFoundError: No module named 'nemo_curator.scripts.fuzzy_deduplication'
```
I think you need an `__init__.py` in the new subfolder you created.